### PR TITLE
Fix typo and improve readability

### PR DESCRIPTION
**Pull request by codeinthehole**

### CODE REVIEW

**Comment by apparentlymart** [comment]
:+1:

File: examples/aws-elb/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-The example launches a web server, installs nginx, creates an ELB for instance. It also creates security groups for elb/instance 
+The example launches a web server, installs nginx, creates an ELB for instance. It also creates security groups for the ELB and EC2 instance. 
 
 To run, configure your AWS provider as described in https://www.terraform.io/docs/providers/aws/index.html
 
-Running the example
+Run this example using:
 
-run `terraform apply -var 'key_name={your_key_name}}'` 
+    terraform apply -var 'key_name={your_key_name}}'
```
Comment (berendt) [inline examples/aws-elb/README.md]: I would prefer to use `YOUR_KEY_NAME` (without `{` and `}`) here. At least the second `}` should be removed.

**Comment by radeksimko** [comment]
Thanks @codeinthehole 

@berendt Agreed, I will fix this in a separate commit.

**Comment by radeksimko** [comment]
https://github.com/hashicorp/terraform/commit/6cf198b70b2835c85f0921889d23e0c798146a58

**Pull request merged**